### ISSUE TITLE
JAWS does not support aria-level on h1-h6

### ISSUE THREAD

**Issue by stevefaulkner** [FreedomScientific/standards-support#814]
### Summary

1. Go to https://stevefaulkner.github.io/AT-browser-tests/test-files/h7-h10.html
2.  3. navigate to heading using the h key

### Expected result

heading level is announced based on aria-level

### Actual result

heading level is announced based on h1-h6 element
JAWS announces headings as "heading level 2" in Firefox

#### JAWS version and build number
JAWS 2024.2312.53 

 
#### Operating System and version

Windows 11

#### Browser and version:

Chrome Version 122.0.6261.58 (Official Build) (64-bit)
Firefox 123.0 (64-bit)
Edge Version 122.0.2365.59 (Official build) (64-bit)

**Comment by stevefaulkner** [comment]
Tested in latest JAWS and latest chrome, still does not work correctly heading level 2's all the way.
